app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -396,7 +451,7 @@ fun DepthButton(
     content: @Composable (BoxScope.() -> Unit),
 ) {
     val contentColor by colors.contentColor(isEnabled)
-    val onClickState = rememberUpdatedState(onClick)
```

Comment:
Please reverse this. This is actually a fix to one terrible bug.

Reply:
Addressed.! Thank you